toolchain/sem_ir/entity_name.h

```diff
@@ -44,7 +47,7 @@ struct EntityName : public Printable<EntityName> {
   // The parent scope.
   NameScopeId parent_scope_id;
 
-  // TODO: The following two fields are only meaningful for a symbolic binding.
```

Comment:
I assume this is what you're calling a typo, but are't there only two following?

Reply:
Oh right, it's going to become three, it's fine for now but not future proof. I will rephrase the PR.

Reply:
"future proof" is relative, in that we could also add a field after this TODO that it does *not* apply to. Or, perhaps the PR you're thinking about adding a third field will change to store that elsewhere as part of implementation/review. `EntityName` is used frequently, and it might be worth considering the overhead if more symbolic-binding-only fields are to be added.

Maybe leave it as-is, and update it as part of the other work as needed?

Reply:
Sure, done